drivers/disk/sdmmc_stm32.c

```diff
@@ -18,12 +18,50 @@
 
 LOG_MODULE_REGISTER(stm32_sdmmc, CONFIG_SDMMC_LOG_LEVEL);
 
+#define STM32_SDMMC_USE_DMA DT_NODE_HAS_PROP(DT_DRV_INST(0), dmas)
+
+#if STM32_SDMMC_USE_DMA
+#include <zephyr/drivers/dma.h>
+#include <zephyr/drivers/dma/dma_stm32.h>
+#include <stm32_ll_dma.h>
+#endif
+
 #ifndef MMC_TypeDef
 #define MMC_TypeDef SDMMC_TypeDef
 #endif
 
+#ifndef SDMMC_BUS_WIDE_1B
+#define SDMMC_BUS_WIDE_1B SDIO_BUS_WIDE_1B
+#endif
+
+#ifndef SDMMC_BUS_WIDE_4B
+#define SDMMC_BUS_WIDE_4B SDIO_BUS_WIDE_4B
+#endif
+
+#ifndef SDMMC_BUS_WIDE_8B
+#define SDMMC_BUS_WIDE_8B SDIO_BUS_WIDE_8B
+#endif
```

Comment:
Seems all this is already in the driver. Can you rebase

Reply:
Yeah. It was in another pull request that I got merged before this. I have now done a rebase.